src/Controller/AdminController.php

```diff
@@ -214,6 +224,38 @@ public function getRefunds(string $orderId): JsonResponse
         return new JsonResponse($this->buildRefundResponseData($refunds->getElements()));
     }
 
+    /**
+     * Get all the notifications for an order.
+     *
+     * @Route(
+     *     "/api/adyen/orders/{orderId}/notifications",
+     *      methods={"GET"}
+ *     )
+     * @param string $orderId
+     * @return JsonResponse
+     */
+    public function getOrderNotifications(string $orderId): JsonResponse
+    {
+        $order = $this->orderRepository->getOrder($orderId, Context::createDefaultContext());
+        $notifications = $this->notificationService->getAllNotificationsByOrderNumber($order->getOrderNumber());
+
+        $response = [];
+        /** @var NotificationEntity $notification */
+        foreach ($notifications as $notification) {
+            $response[] = [
+                'pspReference' => $notification->getPspreference(),
+                'eventCode' => $notification->getEventCode(),
+                'success' => $notification->isSuccess(),
+                'amount' => $notification->getAmountValue() . ' ' . $notification->getAmountCurrency(),
+                'status' => $notification->isDone() ? 'PROCESSED' : 'PENDING',
```

Comment:
Same here, 'PROCESSED' : 'PENDING' could be moved to constants

Reply:
Done